### PR TITLE
[NUI] make RelativeLayout public

### PR DESCRIPTION
**Pull request by dasg34**
### API Changes ###

 - ACR: https://code.sec.samsung.net/jira/browse/TCSACR-392

### CODE REVIEW

File: src/Tizen.NUI/src/public/Layouting/RelativeLayout.cs [hunk 1]
```diff
@@ -23,85 +23,85 @@ namespace Tizen.NUI
...
     /// </summary>
-    [EditorBrowsable(EditorBrowsableState.Never)]
+    /// <since_tizen> 9 </since_tizen>
     public partial class RelativeLayout : LayoutGroup
     {
         /// <summary>
         /// LeftTargetProperty
         /// </summary>
-        [EditorBrowsable(EditorBrowsableState.Never)]
+        /// <since_tizen> 9 </since_tizen>
```
Comment (rabbitfor) [inline src/Tizen.NUI/src/public/Layouting/RelativeLayout.cs:32]: I wonder if we need to open bindable properties.

Comment (dasg34) [reply src/Tizen.NUI/src/public/Layouting/RelativeLayout.cs]: Oh, we don't need to open bindable properties..!
Patch has been updated to remove bindable properties ACR.

Comment (rabbitfor) [reply src/Tizen.NUI/src/public/Layouting/RelativeLayout.cs]: Thanks!

**Comment by dongsug-song** [comment]
@dasg34  
Hello,
please do not upload ACR-PR in DevelNUI branch.
here is a tip to handle ACR-PR here, I hope this helps.
thank you.

- Add-ACR : 
  1) the ACR-PR should be processed in master branch only.
  2) so, make API as hidden by adding `[EditorBrowsable(EditorBrowsableState.Never)]` attribute, then in DevelNUI branch API-Checker will pass. (No "ACR Required" tag)
  3) after the patch is merged into master branch by regular integration work, you can make new PR in master branch by removing `[EditorBrowsable(EditorBrowsableState.Never)]` attribute to proceed ACR.
  4) if the ACR is urgent, you can upload PR on master branch directly. that PR will be merged on DevelNUI branch by rebasing during the regular integration work.

- Deprecate-ACR : 
  1) the ACR-PR should be processed in master branch only.
  2) so, add some mark on the API by writing some comments like `//ToDo : will raise deprecated-ACR, [Obsolete("This has been deprecated in APIxxx and will be removed in APIxxx+2. The xxx variable is not used anymore and the xxx is replaced by xxx! Please use xxx instead!")`
  3) by step 2), the PR on DevelNUI will pass. (No "ACR Required" tag)
  4) after the patch is merged into master branch by regular integration work, you can make new PR in master branch by adding `[Obsolete("blah blah")]` tag in order to proceed ACR.
  5) if the ACR is urgent, you can upload PR on master branch directly. that PR will be merged on DevelNUI branch by rebasing during the regular integration work.

**Comment by dasg34** [comment]
> @dasg34
> Hello,
> please do not upload ACR-PR in DevelNUI branch.
> here is a tip to handle ACR-PR here, I hope this helps.
> thank you.
> 
> * Add-ACR :
>   
>   1. the ACR-PR should be processed in master branch only.
>   2. so, make API as hidden by adding `[EditorBrowsable(EditorBrowsableState.Never)]` attribute, then in DevelNUI branch API-Checker will pass. (No "ACR Required" tag)
>   3. after the patch is merged into master branch by regular integration work, you can make new PR in master branch by removing `[EditorBrowsable(EditorBrowsableState.Never)]` attribute to proceed ACR.
>   4. if the ACR is urgent, you can upload PR on master branch directly. that PR will be merged on DevelNUI branch by rebasing during the regular integration work.
> * Deprecate-ACR :
>   
>   1. the ACR-PR should be processed in master branch only.
>   2. so, add some mark on the API by writing some comments like `//ToDo : will raise deprecated-ACR, [Obsolete("This has been deprecated in APIxxx and will be removed in APIxxx+2. The xxx variable is not used anymore and the xxx is replaced by xxx! Please use xxx instead!")`
>   3. by step 2), the PR on DevelNUI will pass. (No "ACR Required" tag)
>   4. after the patch is merged into master branch by regular integration work, you can make new PR in master branch by adding `[Obsolete("blah blah")]` tag in order to proceed ACR.
>   5. if the ACR is urgent, you can upload PR on master branch directly. that PR will be merged on DevelNUI branch by rebasing during the regular integration work.

This PR is for master branch..

**Comment by dasg34** [comment]
Ping~
Can i merge this patch?

**Comment by dongsug-song** [comment]
@dasg34 
Hello,
sorry for my mistake. you are right.
the ACR (https://code.sec.samsung.net/jira/browse/TCSACR-392) is accepted, so I merge this.
thank you.

**Review by dongsug-song — APPROVED** [review]
looks good to me
ACR has been accepted

**Pull request merged**